packages/tlschema/src/records/TLDocument.ts

```diff
@@ -2,6 +2,9 @@ import { BaseRecord, createRecordType, defineMigrations, RecordId } from '@tldra
 import { JsonObject } from '@tldraw/utils'
 import { T } from '@tldraw/validate'
 
+/** @public */
+export const DEFAULT_DOCUMENT_NAME = 'Untitled'
+
```

Comment:
Not sure we should have this.
I think in the places where we render/use the document name we should probably be using a localized version of 'Untitiled' if the .name property is the empty string.

Reply:
I'll take a look. Think the problem was that our current translations always expect us to get translations via hooks. But I think in one of the cases that wasn't really feasible here.

That said we can definitely remove this default here and always check if it's an empty string where we show it.